### ISSUE TITLE
[Jetfighter] addition of bombs, rockets, and shield

### ISSUE THREAD

**Issue by Trikkii** [Wolfyxon/3ds-web-stuff#35]
Using the touch screen tapping the bottom right corner of the screen to change from the bullets that your already shooting to be bombs, and the middle right part of the screen would be to change to rockets. the top right part of the screen would turn on and off the shield.

**Comment by Wolfyxon** [comment]
Yeah I was planning that, but for now I'm focusing on other games.
